source/pdps-group-replication.rst

```diff
@@ -0,0 +1,233 @@
+
+The node characteristics such as CPU/RAM/Storage are not relevant to the solution design.  They must reflect the estimated workload that the solution will have to cover, and this is a case by case identification.
+
+However, it is important that all nodes that are part of the cluster must have the same characteristics. Otherwise, the cluster is imbalanced and services will be affected.
+
+As a generic indication we recommend using nodes with at least 8 cores and 16GB RAM when in production.
+
+Technical overview
+==================
+.. _ha:
+
+High Availability
+-------------------
+
+How to measure availability and at what point does it become "high" availability?
+
+Generally speaking, the measurement of availability is done by establishing a measurement time frame and dividing it by the time that it was available. This ratio will rarely be 1, which is equal to 100% availability. A solution is considered to be highly available if it is at least 99% or "two nines" available.  
+
+The following table provides downtime calculations per high availability level:
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Availability, %
+     - Downtime per year
+     - Downtime per month
+     - Downtime per week
+     - Downtime per day
+   * - 99% ("two nines")
+     - 3.65 days
+     - 7.31 hours
+     - 1.68 hours
+     - 14.40 minutes
+   * - 99.5% ("two nines five")
+     - 1.83 days
+     - 3.65 hours
+     - 50.40 minutes
+     - 7.20 minutes
+   * - 99.9% ("three nines")
+     - 8.77 hours
+     - 43.83 minutes
+     - 10.08 minutes
+     - 1.44 minutes
+   * - 99.95% ("three nines five")
+     - 4.38 hours
+     - 21.92 minutes
+     - 5.04 minutes
+     - 43.20 seconds
+   * - 99.99% ("four nines")
+     - 52.60 minutes
+     - 4.38 minutes
+     - 1.01 minutes
+     - 8.64 seconds
+   * - 99.995% ("four nines five")
+     - 26.30 minutes
+     - 2.19 minutes
+     - 30.24 seconds
+     - 4.32 seconds
+   * - 99.999% ("five nines")
+     - 5.26 minutes
+     - 26.30 seconds
+     - 6.05 seconds
+     - 864.00 milliseconds
+  
+How is high availability achieved?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are 3 key components to achieve high availability:
+
+1. **Infrastructure** - This is the physical or virtual hardware that database systems rely on to run. Without enough infrastructure (VM’s, networking, etc.) there cannot be high availability. The easiest example is: ``there is no way to make a single server highly available``.
```

Comment:
This should be an unordered list since this is not a procedure. The order of the components does not matter.

The repeated "This is" is redundant. 

infrastructure (VM’s, networking, etc.), there cannot